### ISSUE TITLE
Auto resolve "Xcode can be updated from the App Store"?

### ISSUE THREAD

**Issue by CNG** [mas-cli/mas#84]
I want to use `mas` in a script, but is there no way to install `mas` itself without user intervention or triggering an update of all system software?

```
$ brew install mas
Error: Your Xcode (7.2) is too outdated.
Please update to Xcode 8.3 (or delete it).
Xcode can be updated from the App Store.
```

It seems the options are running `softwareupdate --install --all` (what if I don't want to update macOS, especially if this is a fragile hackintosh?) or asking the user to open the App Store app and click on the update link next to Xcode.

**Comment by rodionovd** [comment]
Hello @CNG, thanks for reaching out!

It seems to me that the issue you're describing is related to Homebrew itself and not `mas`.

Anyway, I believe that you can use `softwareupdate` to upgrade *just* your Xcode application. I'm thinking of something like this:

```
$ softwareupdate -i "Xcode"
```

**Comment by CNG** [comment]
Hey there,

Unfortunately I don't think that command works, and I haven't found any evidence online of anyone else being able to update Xcode by command line. Many references to `softwareupdate` but nobody claiming to have actually done it!

Thanks for the followup though!

**Comment by CNG** [comment]
Oh, I meant to add even `softwareupdate --install --all` does not update Xcode, I found. It seems there's no option but through the GUI.

**Comment by rodionovd** [comment]
On more thing: do you actually *need* Xcode 7? Homebrew offers pre-built versions (bottles) for popular projects (`mas` included) for both 10.11 and 10.12 as far as I remember. So I believe if you remove your Xcode, it will try to fetch those pre-built versions.

**Comment by CNG** [comment]
Ah, is that what the "or delete it" in the error is referring to? I don't actually use Xcode directly for anything, so I'd be happy to have Homebrew use its versions... I will have to try that out later. Thank you!
